cmd/notifiers.go

```diff
@@ -66,12 +68,128 @@ type notifier struct {
 	// Add new notification queues.
 }
 
+// SetAMQPByID -
+func (n *notifier) SetAMQPByID(accountID string, amqpn amqpNotify) {
+	n.Lock()
+	defer n.Unlock()
+	n.AMQP[accountID] = amqpn
+}
+
+func (n *notifier) GetAMQP() map[string]amqpNotify {
+	return n.AMQP
```

Comment:
> Why would you hold a lock ? We are not modifying the members.

Currently, the caller of GetAMQP will iterate over n.AMQP elements, we can get a problem with a concurrent write